src/idex.ts

```diff
@@ -81,28 +114,28 @@ export class IdentityResolver {
       }
 
       const expiresAt = responseExpires(response)
-      successCallback(responseObj, { expiresAt })
+      successCallback(this.enrichExtraIdentifiers(responseObj, params), { expiresAt })
     }
   }
 
-  unsafeResolve(successCallback: (result: unknown, meta: ResolutionMetadata) => void, errorCallback: (e: unknown) => void, additionalParams: ResolutionParams): void {
-    this.calls.ajaxGet(
-      this.getUrl(additionalParams),
-      this.responseReceived(successCallback),
-      errorCallback,
-      this.timeout
-    )
+  private buildUrl(params: [string, string][]): string {
+    return `${this.url}/${this.source}/${this.publisherId}${toParams(this.filterParams(params))}`
   }
 
   getUrl(additionalParams: Record<string, string | string[]>): string {
```

Comment:
Any idea why we expose the `resolutionCallUrl` in LiveConnect at all `getUrl` is used to compute `resolutionCallUrl`?

Reply:
No idea. As you say, it's just an alias for getUrl `resolutionCallUrl: resolver.getUrl.bind(resolver),`